### ISSUE TITLE
No use of BRATS2015_Testing data

### ISSUE THREAD

**Issue by lokeshkvn** [LAB1/Patch-based-3D-FCN-for-BraTS-Tumor-Segmentation#1]
Hello,

I found that with brats 2015 dataset we get testing data as well but here you haven't  use it for testing or validation. How and where to use this testing data?

**Comment by LAB1** [comment]
The ground truth of testing data was not available to the public. Thus, I only split the training data into training and validation set and evaluated the model. To use the testing set, you may need to register the BRATS challenge and submit your prediction on the testing data. Then hopefully the challenge organizer will return the evaluation results of the metrics on testing data (However, I didn't try it). Good luck!

**Comment by kom102945** [comment]
Hello,
In Patch_FCN_32.py,
saver.restore(sess, "./model_32_1209_final/Patch_3dFCN_32.ckpt")
I get an error in this sentence. 
Are checkpoints created within the model? 
Or how do we solve the problem?
